render-tests/raster-extent/maxzoom/style.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 256
+    }
+  },
+  "center": [
+    13.418056,
+    52.499167
+  ],
+  "zoom": 16,
```

Comment:
I would expect this test to "pass" even if the layer's `maxzoom` wasn't being respected because we don't have any z16 satellite tile fixtures https://github.com/mapbox/mapbox-gl-test-suite/tree/master/tiles